src/main/java/com/google/sps/servlets/DataFromDatabaseServlet.java

```diff
@@ -95,71 +105,75 @@ private void initializeDatabase(String databaseName) {
  
       while (resultSet.next()) {
-        Row rowObject = new Row();
+        ImmutableList.Builder<String> rowBuilder = new ImmutableList.Builder<String>();
         for (Schema schema : schemas) {
           String columnName = schema.columnName();
  
           // If there is a null in this col here, just print out NULL for now.
           if (resultSet.isNull(columnName)) {
-            rowObject.addData(columnName, constants.NULL);
+            rowBuilder.add(constants.NULL);
             continue;
           }
  
           String dataType = schema.schemaType();
-          addDataToRowObject(dataType, rowObject, columnName, resultSet);
+          addDataToRow(dataType, rowBuilder, columnName, resultSet);
         }
-        tableObject.addRow(rowObject);
+
+        ImmutableList<String> immutableRow = rowBuilder.build();
+        tableBuilder.addRow(immutableRow);
       }
     }
   }
 
-  //TODO(issue 15): get rid of Row and change to List of Strings
-  private void addDataToRowObject(String dataType, Row rowObject, String columnName, ResultSet resultSet) {
+  private void addDataToRow(String dataType, ImmutableList.Builder<String> rowBuilder, String columnName, ResultSet resultSet) {
     switch (dataType) {
       case "STRING":
-        rowObject.addData(columnName, resultSet.getString(columnName));
+        rowBuilder.add(resultSet.getString(columnName));
         break;
       case "BOOL":
-        rowObject.addData(columnName, constants.EMPTY_STRING + resultSet.getBoolean(columnName));
+        rowBuilder.add(constants.EMPTY_STRING + resultSet.getBoolean(columnName));
         break;
       case "INT64":
-        rowObject.addData(columnName, constants.EMPTY_STRING + resultSet.getLong(columnName));
+        rowBuilder.add(constants.EMPTY_STRING + resultSet.getLong(columnName));
         break;
       case "BYTES":
         String byteToString = bytesToString(resultSet.getBytes(columnName));
-        rowObject.addData(columnName, byteToString);
+        rowBuilder.add(byteToString);
         break;
       case "TIMESTAMP":
-        rowObject.addData(columnName, constants.EMPTY_STRING + resultSet.getTimestamp(columnName));
+        rowBuilder.add(constants.EMPTY_STRING + resultSet.getTimestamp(columnName));
         break;
       case "DATE":
-        rowObject.addData(columnName, constants.EMPTY_STRING + resultSet.getDate(columnName));
+        rowBuilder.add(constants.EMPTY_STRING + resultSet.getDate(columnName));
         break;
       case "ARRAY<INT64>":
         String arrayToString = longArrayToString(resultSet.getLongArray(columnName));
-        rowObject.addData(columnName, arrayToString);
+        rowBuilder.add(arrayToString);
         break;
       default:
-        rowObject.addData(columnName, "This type is not currently supported.");
+        rowBuilder.add(constants.UNSUPPORT_ERROR);
     }
   }
 
-  //TODO(issue 15): change to String Builder
   private String longArrayToString(long[] longArray) {
```

Comment:
There is an awesome method that may help you here - String.join() 

https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-

Reply:
Done!